exercises/resistor-color/resistor-color.test.ts

```diff
@@ -0,0 +1,21 @@
+import { colorCode, COLORS } from './resistor-color'
+
+describe('color code', () => {
+  it('Black', () => {
+    expect(colorCode('black')).toEqual(0)
+  })
+
+  xit('White', () => {
+    expect(colorCode('white')).toEqual(9)
+  })
+
+  xit('Orange', () => {
+    expect(colorCode('orange')).toEqual(3)
+  })
```

Comment:
Noticed that these tests dont cover all the scenarios. no-one for each color... I am not a maintainer so 🤷‍♂️

Reply:
I beg to disagree. Tests need not check all inputs. 

I was following https://github.com/exercism/problem-specifications/blob/master/exercises/resistor-color/canonical-data.json

Reply:
Yeah I am just questioning the status quo 👼

Reply:
Can you give a good argument why each colour should be tested? Please extend the argument for two, three, four and five band inputs.

Reply:
Since we are using tests to guide the person to get to the desired solution, it seems as though test driven development should be taken into consideration. This would mean that every piece of the code would have a test associated to it. 
As a counter though, one could argue that the last test covers this criteria.